lib/rails_edge_test/version.rb

```diff
@@ -1,3 +1,3 @@
 module RailsEdgeTest
-  VERSION = "1.2.3"
+  VERSION = "1.2.4"
```

Comment:
I guess we should bump this to `2.0.0` because we're no longer testing against older versions of ruby, rails